src/restaurant_inspections_api/handlers/inspections.clj

```diff
@@ -6,42 +6,27 @@
 (defn validate-inspections-params
   "Receives all inspections query parameters (nil when not specified) and returns a map of valid and
   invalid params."
-  [zip-codes business-name start-date end-date district-code county-number per-page page]
+  [{zip-codes :zipCodes
```

Comment:
There is a shorthand for this:
```{:keys [zip-codes business-name start-date]}```
(omitted some of the keys but you can destructure all this way)